resources/js/Components/CharacterDetail/CharacterDetailBox.tsx

```diff
@@ -29,6 +31,12 @@ type CharDetailBoxPropType = {
     setIsEditDrawerOpen: (payload: boolean) => void
 }
 
+function getUser() {
```

Comment:
put this as an arrow function and within the component please. if you put it here the component will not re-render on state change